docs/kyma/04-03-cluster-installation.md

```diff
@@ -11,14 +11,42 @@ If you need to use Helm and access Tiller, complete the [additional configuratio
 
 >**CAUTION:** These instructions are valid starting with Kyma 1.2. If you want to install older releases, refer to the respective documentation versions.
 
-Choose your cloud provider and get started:
+Choose the installation type and get started:
 
-<div tabs>
+<div tabs name="provider-installation">
   <details>
   <summary>
-  GKE
+  GCP Marketplace
   </summary>
 
+  1. Access the Google Cloud Platform (GCP) Console at `https://console.cloud.google.com/`.
```

Comment:
Hm... better will be to point to https://console.cloud.google.com/marketplace and then select Project

Reply:
In fact, you don't need to select a project, because it will ask you when you click `Configure`

Reply:
Ok, I just wanted to be a bit in line with the video